server/src/main/java/org/apache/iotdb/db/conf/IoTDBConstant.java

```diff
@@ -60,8 +60,8 @@ private IoTDBConstant() {
   public static final String COLUMN_VALUE = "value";
   public static final String COLUMN_VERSION = "        version";
   public static final String COLUMN_TIMESERIES = "timeseries";
-  public static final String COLUMN_TIMESERIES_DataType = "dataType";
-  public static final String COLUMN_TIMESERIES_Encoding = "encoding";
+  public static final String COLUMN_TIMESERIES_DataType = "DATATYPE";
+  public static final String COLUMN_TIMESERIES_Encoding = "ENCODING";
```

Comment:
Sorry I did not make it clear, I mean COLUMN_TIMESERIES_DataType -> COLUMN_TIMESERIES_DATATYPE and
COLUMN_TIMESERIES_Encoding -> COLUMN_TIMESERIES_ENCODING since static final variables should be in the upper case. And please revert the strings because other column names are still in lower case.